R4ML/R/ml.model.svm.R

```diff
@@ -150,24 +151,24 @@ setMethod("initialize", "r4ml.svm",
 #' @examples \dontrun{
 #' 
 #' # Load the Iris dataset 
-#' iris_hf <- as.r4ml.frame(iris)
+#' iris_r4mlDF <- as.r4ml.frame(iris)
```

Comment:
yes, point taken on the number...but I'm not sure we can change the usage of r4ml throughout the code (in class names, for example).  I'll keep it as r4ml_df for now...